entity_highlights.py

```diff
@@ -0,0 +1,75 @@
+from dbco import *
+from nltk.tokenize import sent_tokenize
+import re
+
+def create_sentences(content):
+	'''Divide content into sentences'''
+	sentences = sent_tokenize(content)
+	valid = []
+	for s in sentences:
+		
+		try:
+			# Decodes sentences names so python can work with them, removes newline characters
+			s = str(s).strip()
+			valid.append(s)
+
+		except UnicodeEncodeError:
+			pass
+
+	return valid
+
+def get_highlighted_sentences(sentences, entities):
+	'''Takes in paragraphs of text and a list of entities, returns a list of the sentences in content in which any number of words in entitites shows'''
+	words_re = re.compile("|".join(entities))
```

Comment:
Delete this line, since the regex is unused. I'm trying to think of clever way of doing regex, but the simplest thing I can think of is to 
```
entsRe = re.compile('|'.join('({})'.format(ent) for ent in entities), re.IGNORECASE)
for sent in sentences:
   matches = entsRe.findall(sentence)
   for match in matches:
       # match the index of the match tuple to the index in the ent list
```

which isn't cleaner than the first thing we have